### ISSUE TITLE
Add a canonical script that installs Knative

### ISSUE THREAD

**Issue by tdiesler** [apache/camel-k#5545]
### Requirement

Installing Knative serving/eventing is a multistep process. It would help to have a common script that can be executed by users and tests.

**Comment by lburgazzoli** [comment]
Relates to https://github.com/apache/camel-k/issues/5463

**Comment by tdiesler** [comment]
```
minikube-delete && minikube-start && make images && kamel install
./e2e/knative/files/setup.sh && minikube tunnel
time go test -v -tags=integration ./e2e/knative --run TestKnativePlatformDetection
```

**Comment by tdiesler** [comment]
PR: https://github.com/apache/camel-k/pull/5573

**Comment by squakez** [comment]
Is this solved by #5573? Can I close or still any leftover?
